xbout/geometries.py

```diff
@@ -75,49 +78,68 @@ def wrapper(add_geometry_coords):
 
 
 @register_geometry('toroidal')
-def add_toroidal_geometry_coords(ds):
+def add_toroidal_geometry_coords(ds, coordinates=None):
+
+    if coordinates is None:
+        coordinates = ('psi', 'theta', 'phi')
+
+    # Check whether coordinates names conflict with variables in ds
+    bad_names = []
+    for name in coordinates:
+        if name in ds:
+            bad_names.append(name)
+    if bad_names:
+        raise ValueError('Coordinate names {} clash with variables in the dataset. '
+                         "Use the 'coordinates' argument of open_boutdataset to provide "
+                         "alternative names".format(bad_names))
 
     # Change names of dimensions to Orthogonal Toroidal ones
-    ds = ds.rename(y='theta', inplace=True)
+    ds = ds.rename(y=coordinates[1])
```

Comment:
If `coordinates` were a dictionary (e.g. `{'x': 'radial', 'y': 'parallel', 'z': 'binormal'}` instead of a list of strings, this would be clearer throughout, and avoid "magic numbers".

Reply:
Makes sense - also allows for nicer defaults, so the user can pass just one or two of `'x'`, `'y'` or `'z'` names instead of being required to pass all of them.

I think the use of `coordinates` argument in the s-alpha case was broken before too - hopefully fixed now.